src/Tinkoff.ISA.Providers.Jira/ServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,47 @@
+using System;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Options;
+using Tinkoff.ISA.Core;
+using AtlassianJira = Atlassian.Jira.Jira;
+
+namespace Tinkoff.ISA.Providers.Jira
+{
+    public static class ServiceCollectionExtensions
+    {
+        public static IServiceCollection AddJiraProvider(this IServiceCollection serviceCollection, IConfiguration configuration)
```

Comment:
instead of IConfiguration can we just create simple our interface or class with specific properties? Because we should create a hard guid to explain how they should create section in appsettings.json may be they will use other mechanism or other logic how to get settings of Jira and it will be more obvious just to send this parameters during setup

Reply:
But doesn't it create a mess in composition root if we bound developers to do all configuration job there? Maybe it becomes more obvious but we lose flexibility, don't we?

Reply:
Ok, I see. You're right